dev-java/jopt-simple/jopt-simple-5.0-r1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="A Java library for parsing command line options"
+HOMEPAGE="https://jopt-simple.github.io/jopt-simple/"
+SRC_URI="https://github.com/jopt-simple/jopt-simple/archive/jopt-simple-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
```

Comment:
Why 0 and not e.g. 5.0 (or maybe even 5)?

Reply:
It has reverse dependencies working well with slot 0.  If you changed the slot you would have to adjust all reverse dependencies.
```
$ qgrep -e 'jopt-simple.*0'
sci-geosciences/bt747/bt747-2.1.7-r2.ebuild:    dev-java/jopt-simple:0
```

Reply:
Right, that was me being stupid. I assumed that this PR introduces SLOT=0 for jopt 5.